Introduce the unit selector menu and sub-widget (#4266)
qgis/QGIS-Documentation: source/docs/user_manual/introduction/general_tools.rst

* and :guilabel:`Paste Format` to the text, speeding configuration.

.. index:: Unit selection; Map scale
.. _unit_selector:

Unit Selector
--------------

> Size properties of the items (labels, symbols, layout elements...) in QGIS are not

Reviewer: ```suggestion
Size properties of the items (labels, symbols, layout elements, ...) in QGIS are not
```

> necessarily bound to either the project units or a particular layer units.

Reviewer: ```suggestion
necessarily bound to either the project units or the units of a particular layer.
```

For a large set of properties, the :guilabel:`Unit` selector drop-down menu
allows you to tweak their value according to the rendering you want (based on
screen resolution, paper size, or the terrain). Available units are:

* :guilabel:`Millimeters`
* :guilabel:`Points`
* :guilabel:`Pixels`

> * :guilabel:`Meters at Scale`: This allows setting the size always in meters,

Reviewer: ```suggestion
* :guilabel:`Meters at Scale`: This allows you to always set the size in meters,
```

  regardless of what the underlying map units are (e.g. they can be in inches, feet,

>   geographic degrees). The size in meters is calculated based on the current project

Reviewer: ```suggestion
  geographic degrees, ...). The size in meters is calculated based on the current project
```

  ellipsoid setting and a projection of the distances in meters at the center of the
  current map extent.
* :guilabel:`Map Units`: The size is scaled according to the map view scale.
  Because this can lead to too big or too small values, use the |options| button

>   next to the entry to constrain the size to apply in a range of values based on:

Reviewer: ```suggestion
  next to the entry to constrain the size to a range of values based on:
```
